scripts/upgrade

```diff
@@ -12,81 +12,59 @@ source /usr/share/yunohost/helpers
-botadmin=$(ynh_app_setting_get --app=$app --key=botadmin)
 botusers=$(ynh_app_setting_get --app=$app --key=botusers)
-mautrix_config_path="$final_path/config.yaml"
-mautrix_version=$(ynh_app_setting_get --app=$app --key=mautrix_version)
+botadmin=$(ynh_app_setting_get --app=$app --key=botadmin)
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+previous_mautrix_version=$(ynh_app_setting_get --app=$app --key=mautrix_version)
+
+db_name=$(ynh_app_setting_get --app=$app --key=db_name)
+# Upgrade from <=0.2.0
+db_pwd=$(ynh_app_setting_get --app=$app --key=mautrix_bridge_db_pwd)
+if [$db_pwd == ""]
+then
+# Upgrade from >=0.2.2
+db_pwd=$(ynh_app_setting_get --app=$app --key=db_pwd)
+fi
 
-ynh_print_OFF
-mautrix_bridge_db_pwd=$(ynh_app_setting_get --app=$app --key=mautrix_bridge_db_pwd)
-ynh_print_ON
+mautrix_version=$(ynh_app_upstream_version)
 
 #=================================================
 # SET CONSTANTS
 #=================================================
 
-log_filename="/var/log/$app/$app.log"
+enable_relaybot=true
+# https://docs.python.org/3.6/library/logging.html#logging-levels
+log_level="INFO"
+#log_format="$app.log"
+
 synapse_db_name="matrix_$synapse_instance"
-# botname_synapse_db_user="@$botname:$server_name"
-mautrix_bridge_user=$app
-mautrix_bridge_db_name=$app
-mautrix_bridge_db_user=$app
-upstream_version=$(ynh_app_upstream_version)
+bot_synapse_db_user="@$botname:$server_name"
 
 #=================================================
 # CHECK VERSION
 #=================================================
 
-### This helper will compare the version of the currently installed app and the version of the upstream package.
-### $upgrade_type can have 2 different values
-### - UPGRADE_APP if the upstream app version has changed
-### - UPGRADE_PACKAGE if only the YunoHost package has changed
-### ynh_check_app_version_changed will stop the upgrade if the app is up to date.
-### UPGRADE_APP should be used to upgrade the core app only if there's an upgrade to do.
 upgrade_type=$(ynh_check_app_version_changed)
 
 #=================================================
 # ENSURE DOWNWARD COMPATIBILITY
 #=================================================
-ynh_script_progression --message="Ensuring downward compatibility..." --time --weight=1
-
-#
-# N.B. : the followings setting migrations snippets are provided as *EXAMPLES*
-# of what you may want to do in some cases (e.g. a setting was not defined on
-# some legacy installs and you therefore want to initiaze stuff during upgrade)
-#
-
-# If db_name doesn't exist, create it
-#if [ -z "$db_name" ]; then
-#	db_name=$(ynh_sanitize_dbid --db_name=$app)
-#	ynh_app_setting_set --app=$app --key=db_name --value=$db_name
-#fi
-
-# If final_path doesn't exist, create it
-#if [ -z "$final_path" ]; then
-#	final_path=/var/www/$app
-#	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
-#fi
+ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
```

Comment:
Is this useful at all?

Reply:
We should probably move this
```
# Upgrade from <=0.2.0
db_pwd=$(ynh_app_setting_get --app=$app --key=mautrix_bridge_db_pwd)
if [$db_pwd == ""]
then
# Upgrade from >=0.2.2
db_pwd=$(ynh_app_setting_get --app=$app --key=db_pwd)
fi
```
here. There are regularly some version specific actions to put here